csharp/Facebook.Yoga.sln

```diff
@@ -5,42 +5,124 @@ VisualStudioVersion = 14.0.25420.1
 MinimumVisualStudioVersion = 10.0.40219.1
```

Comment:
This solution contains all projects including both the universal and desktop tests.